src/i18n/en.js

```diff
@@ -12,7 +12,7 @@ const urlShopify = process.env.REACT_APP_SHOPIFY_URL;
 // Common URLs
 const urlHelpAdvancedReports = `${urlHelp}/advanced-reports`;
 const urlPrivacy = `${urlSite}/en/legal/privacy`;
-const urlPrivacyFromSignup = `${urlPrivacy}?utm_source=app&utm_medium=landing&utm_campaign=signup`;
+const urlPrivacyFromSignup = `${urlPrivacy}?utm_source=app&amp;utm_medium=landing&amp;utm_campaign=signup`;
 const urlPrivacyFromLogin = `${urlPrivacy}?utm_source=app&utm_medium=landing&utm_campaign=login`;
```

Comment:
I am not sure if it is a good idea encoding the value of `urlPrivacyFromSignup` because it is only a string, it is not part of HTML yet, in the other hand, notice that `urlPrivacyFromLogin` is not encoded.

Reply:
Because it's use in the footer with Markdown

Reply:
I know, but we cannot change the value depending on who uses the variable.

Imagine that someone wants to also use this variable in markdown content.

Reply:
you suggest use in the key?

Reply:
![image](https://user-images.githubusercontent.com/8861133/62970122-27899800-bde5-11e9-8bc0-5fffe7e2d7ef.png)
like this?

Reply:
No, maybe one of these options (or another):

* Using a sufix

    ```js
     const urlPrivacyFromSignup_HTMLEncoded = `${urlPrivacy}?utm_source=app&amp;utm_medium=landing&amp;utm_campaign=signup`;
    ```

* Converting it before including in the key. 

    ```js
    const urlPrivacyFromSignup = `${urlPrivacy}?utm_source=app&utm_medium=landing&utm_campaign=signup`;
     const urlPrivacyFromSignup_HTMLEncoded = htmlEncode(urlPrivacyFromSignup);
    ```

    or

    ```js
    const prueba_HTML = `<a target="_blank" href="${htmlEncode(urlPrivacyFromSignup)}" . . .`
    ```

* Changing it by markdown

* Other option?

Reply:
changed